### PR TITLE
Codewars

### LINKED ISSUE #97: Add Codewars to RailsSchool

### PR DESCRIPTION
**Pull request by nbhartiya**
This pull request fixes #97, allowing teachers to assign Codewars challenges to students and monitor whether or not they have completed them.

### CODE REVIEW

**Comment by nbhartiya** [comment]
@gkop, I reworked this a little bit. would you mind reviewing again?

File: app/models/user.rb [hunk 1]
```diff
@@ -63,4 +65,12 @@ def badges
       Badge.find_by_badge_id(user_badge.badge_id).new
     }
   end
+
+  def completed_codewar_for_lesson(lesson)
+    codewars.each do |codewar|
+      return true if codewar.slug==lesson.codewars_challenge_slug
+    end
+    false
```
Comment (gkop) [inline app/models/user.rb]:
```ruby
codewars.find_by_slug(lesson.codewars_challenge_slug).present?
```

Comment (gkop) [inline app/models/user.rb]: Also what about checking the completedLanguages?

Comment (nbhartiya) [inline app/models/user.rb]: added an issue for that, they just got the api working properly so will go ahead and add a column for language

**Comment by gkop** [comment]
+1

**Pull request merged**